### ISSUE TITLE
Initial snapshot when PersistenceMode is ManualSnapshots

### ISSUE THREAD

**Issue by loofbon** [DevrexLabs/OrigoDB#24]
Even when PersistenceMode is set to ManualSnapshots, an initial journal entry is still created on disk. Not sure if there’s any way to avoid/defer this for cases when the database is intended to be ephemeral.

**Comment by rofr** [comment]
Good catch, that shouldn't be the case. This is not a fully explored feature, neither in tests nor production use cases. After a quick peek at the code, this looks like it will be a fairly simple change. Meanwhile, if you want to turn off persistence altogether you can use the in memory storage components:

https://github.com/DevrexLabs/OrigoDB/blob/dev/src/OrigoDB.Core/Configuration/ConfigurationExtensions.cs#L14-L19
